helm/kong-app/values.yaml

```diff
@@ -346,6 +346,12 @@ postgresql:
   service:
     port: 5432
 
+  external: 
```

Comment:
Is this only for testing this template?

Reply:
Sorry; I mean password and host in general.

Reply:
Yes, I think it's just for testing. But maybe have a comment here?

When deployed the kong app CR will have a user configmap and secret with the password that sets these values.